sdk/include/psdk/inputscope.idl

```diff
@@ -74,7 +74,7 @@ typedef enum {
 
 cpp_quote("HRESULT WINAPI SetInputScope(HWND hwnd,InputScope inputscope);")
 cpp_quote("HRESULT WINAPI SetInputScopes(HWND hwnd,const InputScope *pInputScopes,UINT cInputScopes,WCHAR **ppszPhraseList,UINT cPhrases,WCHAR *pszRegExp,WCHAR *pszSRGS);")
-cpp_quote("HRESULT WINAPI SetInputScopeXML(HWND hwnd,WCHAR *pszXML);")
+cpp_quote("EXTERN_C HRESULT WINAPI SetInputScopeXML(_In_opt_ HWND hwnd, _In_opt_ PCWSTR pszXML);")
```

Comment:
The EXTERN_C shouldn't be needed.
There should be a global `#ifdef __cplusplus extern "C" #endif` at the top of the file (in a cpp_quote), but it seems missing. This is the case in any of the Win2003, Win7 and Win10 PSDK inputscope.idl.
See for example:
https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/um/InputScope.idl#L31
https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/um/InputScope.h#L182

Reply:
Fixed.